src/lint/rules/algorithm-line-endings.ts

```diff
@@ -0,0 +1,203 @@
+import type { Node as EcmarkdownNode, Observer } from 'ecmarkdown';
+import type { Reporter } from '../algorithm-error-reporter-type';
+
+export default function (report: Reporter, node: Element): Observer {
```

Comment:
You should summarise the behaviour of this rule in a comment here or in an adjacent markdown document.

Reply:
Done.